packages/tool-cache/RELEASES.md

```diff
@@ -1,5 +1,9 @@
 # @actions/tool-cache Releases
 
+### 1.3.1
+
+- Increase http-client min version
```

Comment:
We should specify the PR, even if it includes other changes: https://github.com/actions/toolkit/pull/314